src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -603,25 +701,38 @@ protected override LocalState AllBitsSet()
             return new LocalState(BitVector.Create(nextVariableSlot), BitVector.Create(nextVariableSlot));
         }
 
-        private void EnterParameters(ImmutableArray<ParameterSymbol> parameters)
+        private void EnterParameters()
         {
+            var methodParameters = ((MethodSymbol)_member).Parameters;
+            var signatureParameters = (_methodSignatureUse & MethodSignatureUse.ParameterTypes) == 0 ? default : _methodSignatureOpt.Parameters;
+            int n = methodParameters.Length;
             // label out parameters as not assigned.
-            foreach (var parameter in parameters)
+            for (int i = 0; i < n; i++)
```

Comment:
Comment belongs in next method.

Reply:
Removed.

---
In reply to: [193195590](https://github.com/dotnet/roslyn/pull/27283#discussion_r193195590) [](ancestors = 193195590)